### PR TITLE
Move bazel build tarball test to prow

### PR DESCRIPTION
**Pull request by sallyom**
CORS-751

This PR moves the current travis test 'bazel build tarball' to Prow

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,10 +22,10 @@ jobs:
       name: Terraform format
     - script: ./hack/gotest.sh
       name: Installer unit tests
+    - stage: Build
```
Comment (wking) [inline .travis.yml]: Shifting the `- Stage` entry in this sequence (accidentally?) moves the Go vet and lint jobs from the "Lint & Test" stage to the "Build" stage (e.g. [here][1]).  I think we only need to update the `script: ...` line.

[1]: https://travis-ci.org/openshift/installer/builds/413861629

Comment (sallyom) [reply .travis.yml]: yea, accidentally..

**Pull request merged**